components/ai_chat/resources/page/state/data-context-provider.tsx

```diff
@@ -313,6 +312,10 @@ function DataContextProvider (props: DataContextProviderProps) {
     if (shouldDisableUserInput) return
     if (handleFilterActivation()) return
 
+    if (conversationHistory.length === 0) {
+      getPageHandlerInstance().pageHandler.setShouldSendPageContents(shouldSendPageContents)
+    }
+
```

Comment:
I don't love the 2x calls to the remote. Can't this be combined with `submitConversationEntry` or handled implicitly in the backend?

Reply:
bump